Update custom-applications-index.adoc (#1725)
bonitasoft/bonita-doc: modules/ROOT/pages/custom-applications-index.adoc

= Custom applications
:description: This page explains what an application is, what it is made of, and how you can create your own applications.

{description}

An Application is a navigation of pages that can be accessed by a xref:profiles-overview.adoc[Profile]. +
It is created and operated within Bonita Platform: created with Bonita Studio and UI Designer, deployed with Bonita Continuous Delivery, run by Bonita Runtime.

Its purpose is to display and structure the right information and provide the right controls at the right time to the profiles, so they can efficiently perform their tasks. +

> _Graphically_, an application is made of a navigation menu and pages. In some of the pages, some controls, like -"Validate" or "Reject" buttons- are directly linked to processes and execute tasks in the background. Others -like "New request" or "Add additional information"- open forms that, once submitted start a new process instance, or execute a task. +

Reviewer: ```suggestion
_Graphically_, an application is made of a navigation menu and a set of pages. In some of the pages, some controls, like -"Validate" or "Reject" buttons- are directly linked to processes and execute tasks in the background. Others -like "New request" or "Add additional information"- open forms that, once submitted start a new process instance, or execute a task. +
```

Bonita allows you to create xref:bonita-purpose.adoc#_what_can_i_do_with_bonita[different types of applications]. +

_Structurally_, an application is made of various xref:project-structure.adoc[elements]. Some are directly linked to the application (name, token, navigation), others are referenced (layout, pages, profiles, theme). They are all assembled in the application descriptor. +

The application can be put together by the citizen developer or a professional developer. +

> You just need to make sure that all the elements have been developed before you call them in the application descriptor. +

Reviewer: ```suggestion
You just need to make sure that all the elements have been developed before you can use them in the application descriptor. +
```

--
xref:pages.adoc[[.card-title]#Pages# [.card-body.card-content-overflow]#pass:q[The foundation for a great end-user experience]#]
--

[.card.card-index]
--
xref:profiles.adoc[[.card-title]#Profiles# [.card-body.card-content-overflow]#pass:q[Entities of the organization who can access the application]#]
--

[.card.card-index]
--
xref:layout-development.adoc[[.card-title]#Layout# [.card-body.card-content-overflow]#pass:q[The visual structure of the application]#]
--

[.card.card-index]
--
xref:customize-living-application-theme.adoc[[.card-title]#Theme# [.card-body.card-content-overflow]#pass:q[The graphical details that create a visual identity]#]
--

[.card.card-index]
--
xref:navigation.adoc[[.card-title]#Navigation between applications# [.card-body.card-content-overflow]#pass:q[How to switch from one application to another]#]
--

[.card.card-index]
--
xref:multi-language-applications.adoc[[.card-title]#Multi-language applications# [.card-body.card-content-overflow]#pass:q[How to translate applications in several languages]#]
--

[.card-section]
== The resources 

Starting the development of an application from scratch may not be the most efficient way to go:

> * If your project implies that user picks tasks in a tasklist made of all processes, you may want to try and use or customize our *Bonita User Application*

Reviewer: ```suggestion
* If your project implies that a user picks tasks in a tasklist made of all processes, you may want to try and use or customize our *Bonita User Application*
```
or
```suggestion
* If your project implies that users pick tasks in a tasklist made of all processes, you may want to try and use or customize our *Bonita User Application*
```
